<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ONF_ActionSelectCellsInGrid3DOptions</class>
 <widget class="QWidget" name="ONF_ActionSelectCellsInGrid3DOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="wd_toolbar" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="wd_main" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pb_2d3d">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>52</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/interrupteur_2D.png</normaloff>
             <normalon>:/icons/interrupteur_3D.png</normalon>:/icons/interrupteur_2D.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>52</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_add">
           <property name="toolTip">
            <string>Switch in &quot;add to selection&quot; mode (shortcut : a)</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_colonize">
           <property name="toolTip">
            <string>Switch in &quot;colonize adjacent and not empty values&quot; mode (shortcut : z)</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/colonize.png</normaloff>:/icons/colonize.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_extends">
           <property name="toolTip">
            <string>Switch in &quot;Extends from selected cell&quot; mode (shortcut : e)</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/extends.png</normaloff>:/icons/extends.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_remove">
           <property name="toolTip">
            <string>Switch in &quot;remove from selection&quot; mode (shortcut : r)</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/moins.png</normaloff>:/icons/moins.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_freemove">
           <property name="toolTip">
            <string>Switch in &quot;free move&quot; mode (shortcut : f)</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/move.png</normaloff>:/icons/move.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_centerOnCell">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/center_on_cell.png</normaloff>:/icons/center_on_cell.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_showselected">
           <property name="toolTip">
            <string>Only shows selected cells</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/show_all.png</normaloff>
             <normalon>:/icons/show_selected.png</normalon>:/icons/show_all.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_colonizeParam">
           <property name="maximumSize">
            <size>
             <width>48</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/colonize_param.png</normaloff>:/icons/colonize_param.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wd_2D" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lb_level">
           <property name="toolTip">
            <string>CRTL Mousewheel</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>:/icons/level.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sb_level"/>
         </item>
         <item>
          <widget class="QLabel" name="lb_thickness">
           <property name="toolTip">
            <string>SHIFT Mousewheel</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>:/icons/thickness.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sb_thickness">
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_all">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="toolTip">
            <string>Set maximum level and thickness</string>
           </property>
           <property name="text">
            <string>max</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_cumulMode"/>
         </item>
         <item>
          <widget class="QPushButton" name="pb_copyupwards">
           <property name="toolTip">
            <string>Copy active level selection to all visible levels (see thickness)</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/copy_upwards.png</normaloff>:/icons/copy_upwards.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_copyfull">
           <property name="toolTip">
            <string>Copy active level selection to ALL levels of the grid</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/copy_all.png</normaloff>:/icons/copy_all.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_selectNotEmptyCells">
           <property name="text">
            <string>Select &gt; 0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_convexHull">
           <property name="text">
            <string>Convex</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_gridon">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/grid_on.png</normaloff>
             <normalon>:/icons/grid_off.png</normalon>:/icons/grid_on.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_relative">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>%2D</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wd_3D" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lb_coef">
           <property name="text">
            <string>Coef.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="sbd_coef">
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>0.900000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_gridColor">
           <property name="maximumSize">
            <size>
             <width>38</width>
             <height>25</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
